lynx/common/actions/common_requirements.py

```diff
@@ -5,9 +5,25 @@
 
 class CommonRequirements:
 
-    @staticmethod
-    def is_walkable(scene: 'Scene', object_id: int, movement: Vector) -> bool:
-        object: Object = scene.get_object_by_id(object_id)
-        destination_position: Vector = object.position + movement
-        square_at_destination: Square = scene.get_square(destination_position)
-        return square_at_destination.walkable()
+	@staticmethod
+	def is_walkable(scene: 'Scene', object_id: int, movement: Vector) -> bool:
+		object: Object = scene.get_object_by_id(object_id)
+		destination_position: Vector = object.position + movement
+		square_at_destination: Square = scene.get_square(destination_position)
+		return square_at_destination.walkable()
+
+	@staticmethod
+	def does_object_exists_on_square(scene: 'Scene', position: Vector, name: str) -> bool:
+		objects_on_square = scene.get_objects_by_position(position)
+		for object in objects_on_square:
```

Comment:
You can also use list comprehension, e.g. `return name in [object.name for object in objects_on_square]` (but this is jus a suggestion :) )